Makefile

```diff
@@ -30,9 +30,10 @@ generate-external-crds: ## Generate CRDs for externally defined CRDs and copy th
 	@cd ./control-plane/config/crd/external; \
 		kustomize build | yq --split-exp '.metadata.name + ".yaml"' --no-doc
 
+
 .PHONY: bats-tests
 bats-tests: ## Run Helm chart bats tests.
-	 bats --jobs 4 charts/consul/test/unit
+	docker run -it -v $(CURDIR):/consul-k8s hashicorpdev/consul-helm-test:latest bats --jobs 4 /consul-k8s/charts/consul/test/unit -f "$(TEST_NAME)"
```

Comment:
```suggestion
	docker run -it -v $(CURDIR):/consul-k8s hashicorpdev/consul-helm-test:latest bats --jobs 4 /consul-k8s/charts/consul/test/unit
```

Reply:
I did not think about this deeply, I thought you had may be added this for debugging, but I do see you can pass `-f ""` and it will run the whole suite.  nice.

Reply:
no worries! I figured while I was in here I'd make a small QoL improvement to make this easier to use